RecoLocalCalo/Configuration/python/ecalLocalRecoSequence_cff.py

```diff
@@ -39,7 +41,7 @@
 
 ecalOnlyLocalRecoTask = cms.Task(
     ecalUncalibRecHitTask,
-    ecalRecHitNoTPTask)
+   ecalRecHitNoTPTask)
```

Comment:
remove che space?

Reply:
Yes sorry it's a refuse, I'll restore it